elements/bulbs-dfp/bulbs-dfp.js

```diff
@@ -32,7 +32,7 @@ export default class BulbsDfp extends BulbsHTMLElement {
 
     let defaultRefreshInterval = 30000;
     this.adUnitData = this.adsManager.adUnits.units[this.dataset.adUnit];
-    if (!this.adUnitData.refreshDisabled) {
+    if (!this.adUnitData.refreshDisabled && !this.hasAttribute('no-refresh')) {
```

Comment:
Should be right.  You only want to set up the refresh if it both does not have refresh disabled by the ad unit and does not have the `no-refresh` attribute.  There's test coverage for each scenario too.